digicampipe/scripts/pipeline.py

```diff
@@ -26,11 +26,16 @@
                               [Default: 15]
   --parameters=FILE           Calibration parameters file path
   --template=FILE             Pulse template file path
+  --burst=FILE                File with burst events. If none, no bursts are
+                              considered.
+                              [Default: none]
```

Comment:
Do I understand correctly? If I know beforehand which events are burst events, I can tell the pipeline about it and so they will be ignored?

Reply:
In the monitoring pipeline, I call get_burst.py which save a text file including the range of events in bursts. The path to that file can be passed as --burst argument of the pipeline to skip those event in the analysis.